scripts/fund-contract.js

```diff
@@ -7,16 +7,16 @@ const LinkTokenInterface = artifacts.require('LinkTokenInterface')
   contract for ease-of-use. Any extra LINK present on the contract
   can be retrieved by calling the withdrawLink() function.
 */
-
-const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '1000000000000000000'
+//const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '1000000000000000000'
+const payment = process.env.TRUFFLE_CL_BOX_PAYMENT || '10'
 
 module.exports = async callback => {
   try {
-    const mc = await MyContract.deployed()
-    const tokenAddress = await mc.getChainlinkToken()
+    const mc = '0x162f637509672A6bD993D63c1a21F8d310971505' // your address
+    const tokenAddress = '0x761429d8887600dacE56BfA13651C018C1F0C538' //address of the deploy MyContract
```

Comment:
Having to change a deployment script's source is not user friendly. Can we make these environment variables instead that also take into consideration the deployed contract?